### PR TITLE
Pre commit all the things

### PR DESCRIPTION
**Pull request by pxeger**

### CODE REVIEW

**Review by chunkybanana — APPROVED** [review]

**Comment by pxeger** [comment]
Please don't approve draft pull requests. I marked it as draft because I've not finished making changes, and I'll just have to remove and re-request your review later.

**Review by Lyxal — DISMISSED** [review]
Lgtm but does it lint to line length 80?

**Comment by pxeger** [comment]
It still follows the Black config in `pyproject.toml`

**Comment by Lyxal** [comment]
> It still follows the Black config in `pyproject.toml`

Okay good. Just making sure (also, I didn't see the request not to review until after I'd reviewed and before it was marked as draft iirc)

**Comment by pxeger** [comment]
FYI, what I'm working on is getting the files generated from `elements.yaml` generated and checked by pre-commit.ci

**Comment by pxeger** [comment]
> > It still follows the Black config in `pyproject.toml`
> 
> Okay good. Just making sure (also, I didn't see the request not to review until after I'd reviewed and before it was marked as draft iirc)

> before it was marked as draft

I created it as a draft. But you probably did review before seeing that, so maybe I *won't* throw you to the lions just yet,

**Comment by Lyxal** [comment]
> > > It still follows the Black config in `pyproject.toml`
> > 
> > 
> > Okay good. Just making sure (also, I didn't see the request not to review until after I'd reviewed and before it was marked as draft iirc)
> 
> > before it was marked as draft
> 
> I created it as a draft. But you probably did review before seeing that, so maybe I _won't_ throw you to the lions just yet,

:D I get to live another day!

File: documents/knowledge/yaml_to_txt.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 CURR_DIR = os.path.dirname(os.path.realpath(__file__))
 ELEMENTS_YAML = os.path.join(CURR_DIR, "elements.yaml")
-ELEMENTS_TXT = os.path.join(CURR_DIR, "elements.txt")
+ELEMENTS_TXT = os.path.join(CURR_DIR, "Elements.txt")
```
Comment (pxeger) [inline documents/knowledge/yaml_to_txt.py:6]: Would it be better to just rename `Elements.txt` to be lowercase across the board, for consistency with the others?

Comment (Lyxal) [reply documents/knowledge/yaml_to_txt.py:6]: Renaming it would be worth it

**Review by Lyxal — APPROVED** [review]

File: .github/workflows/linter.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # This workflow will install Python dependencies, run tests and lint with a variety of Python versions
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
 
-name: linter
+name: Run linter
```
Comment (pxeger) [inline .github/workflows/linter.yaml:4]: The inconsistency was annoying me

**Review by cgccuser — APPROVED** [review]
LGTM (and sorry about renaming the linter workflow)

File: documents/knowledge/yaml_to_txt.py (hunk 1 above)
Comment (chunkybanana) [reply documents/knowledge/yaml_to_txt.py]: Yes - several things link to `elements.txt` which doesn't exist

**Pull request merged**